tests/s3/test_bucket_location.py

```diff
@@ -136,13 +137,12 @@ def test_get_bkt_loc_bkt_not_present_273(self):
             assert_utils.assert_false(resp[0], resp[1])
         except CTException as error:
             self.log.info(error)
-            assert_utils.assert_in("NoSuchBucket", str(
-                error.message), error.message)
-        self.log.info(
-            "Step 1 : Get bucket location on non existing bucket failed with error %s",
-            "NoSuchBucket")
-        self.log.info(
-            "Verify get bucket location for the bucket which is not present")
+            assert_utils.assert_in(errconst.NO_BUCKET_OBJ_ERR_KEY,
+                str(error.message), error.message)
```

Comment:
Indentation needs to be aligned, here is an example -
```suggestion
            assert_utils.assert_in(errconst.NO_BUCKET_OBJ_ERR_KEY,
                                   str(error.message), error.message)
```
There are other options as well, like what was original present at lines 141-145

This needs to be done in the block from 140 - 145